spec/datasource_spec.js

```diff
@@ -0,0 +1,170 @@
+import {LightStepDatasource} from "../lightstep-datasource/datasource";
```

Comment:
THIS IS SO GREAT, thank you!

Reply:
It is! I'll add a few more tests for `$__range` and `$__interval`